src/core/net/ip6_mpl.cpp

```diff
@@ -68,7 +68,9 @@ void Mpl::InitOption(OptionMpl &aOption, const Address &aAddress)
     }
 }
 
-ThreadError Mpl::ProcessOption(const Message &aMessage, const Address &aAddress)
+#include <assert.h>
```

Comment:
Why was `<assert.h>` include here? If necessary, can you include `<common/debug.hpp>` instead?